convene-web/spec/requests/spaces/invitations_request_spec.rb

```diff
@@ -16,9 +16,9 @@
 
     # TODO finish building logic to get below assertions to pass
     # generate mailer and views
-    expect(SpaceInvitationMailer).to have_received(:space_invitation_email).with(invitation)
-    expect(mail).to have_received(:deliver_now)
-    expect(invitation.status).to eq("sent")
+    expect(SpaceInvitationMailer).to have_received(:space_invitation_email)
+    expect(mail).to have_received(:deliver_later)
+    expect(invitation.status).to eq("pending")
```

Comment:
@cjoulain - I'm curious about the removal of the `.with(invitation)` constraint in the assertion? It feels useful to confirm that the Mailer was given the correct invitation to deliver.

I'm also curious about the change from confirming the Invitation status was set to 'sent' to 'pending'; as it seems like we would want the invitation to be sent upon creation in this particular case. 

Love the change to expecting this to be delivered later; I'm not entirely sure we have ActiveJob configured yet; so we may want to create a ticket in the Contributor Experience milestone for setting up background jobs; otherwise it's possible these emails won't deliver in production